### PR TITLE
when querying similar suggestions, limit also by days of week

### PR DESCRIPTION
**Pull request by xkjyeah**

### CODE REVIEW

File: src/lib/endpoints/suggestionsWeb.js [hunk 1]
```diff
@@ -537,6 +541,10 @@ export function register(server, options, next) {
           ? `"createdAt" >= '${request.query.createdSince.toISOString()}'::timestamptz`
           : `1=1`
 
+        const daysOfWeekQuery = request.query.daysOfWeek
+          ? `("daysOfWeek" & ${request.query.daysOfWeek}) <> 0`
```
Comment (LoneRifle) [inline src/lib/endpoints/suggestionsWeb.js]: `!=` ?

**Comment by xkjyeah** [comment]
Both are acceptable SQL

**Comment by LoneRifle** [comment]
```
/home/travis/build/datagovsg/beeline-server/test/common.js
   1:49  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style
  # repeat until fade...
```

I'm pretty sure this is your doing =)

File: src/lib/endpoints/suggestionsWeb.js [hunk 2]
```diff
@@ -513,6 +513,10 @@ export function register(server, options, next) {
           time: Joi.number().optional(),
           maxTimeDifference: Joi.number().default(1800e3),
           includeAnonymous: Joi.boolean().default(true),
+          daysOfWeek: Joi.number()
```
Comment (LoneRifle) [inline src/lib/endpoints/suggestionsWeb.js]: This really is `daysMask`, the internal bitmask representation of the suggested days of the week the route should run. Given that how the confusion between this and the virtual Sequelize property `daysOfWeek` led to the CI build breaking, I strongly suggest that you change this

**Comment by xkjyeah** [comment]
Awww you changed it for me. Thanks!

**Pull request merged**